addons/website_sale/views/templates.xml

```diff
@@ -1381,102 +1369,90 @@
-                            <t t-if="line.discount">
-                                <del t-attf-class="#{'text-danger mr8'}" style="white-space: nowrap;" t-esc="combination_info['list_price']" t-options="{'widget': 'monetary', 'display_currency': website_sale_order.currency_id}" />
                             </t>
-                            <small t-if="not line._is_not_sellable_line() and line.product_id.base_unit_price"
-                                class="cart_product_base_unit_price d-block text-muted"
-                                groups="website_sale.group_show_uom_price">
-                                <t t-call='website_sale.base_unit_price'>
-                                    <t t-set='product' t-value='line.product_id'/>
-                                </t>
-                            </small>
-                        </td>
-                        <td class="td-action">
-                            <a href='#' aria-label="Remove from cart" title="Remove from cart" class='js_delete_product no-decoration'> <small><i class='fa fa-trash-o'></i></small></a>
-                        </td>
-                    </tr>
-                </t>
-            </tbody>
+                            <t t-else="">
+                                <span class="text-muted w-100" t-esc="int(line.product_uom_qty)"/>
+                                <input type="hidden" class="js_quantity form-control quantity" t-att-data-line-id="line.id" t-att-data-product-id="line.product_id.id" t-att-value="line.product_uom_qty" />
+                            </t>
+                        </div>
+                    </td>
+                </tr>
+                <tr>
+                    <td t-attf-class="align-bottom pt-0 {{ 'border-bottom pb-4' if not line_last else '' }}">
+                        <a class="js_delete_product d-none d-md-inline-block" href='#' aria-label="Remove from cart" title="Remove from cart"><small>Remove</small></a>
+                        <button type="button" role="button" class="js_delete_product btn btn-light d-inline-block d-md-none " title="Add to Wishlist" data-action="o_wishlist"><span class="fa fa-trash-o" role="img" aria-label="Add to wishlist"></span></button>
+                    </td>
+                    <td t-attf-class="td-price h6 fw-bold text-end pt-0 pe-0 align-bottom {{ 'border-bottom pb-4' if not line_last else '' }}" data-cell-type="price">
+                        <t t-set="combination" t-value="line.product_id.product_template_attribute_value_ids + line.product_no_variant_attribute_value_ids"/>
+                        <t t-set="combination_info" t-value="line.product_id.product_tmpl_id._get_combination_info(combination, pricelist=website_sale_order.pricelist_id, add_qty=line.product_uom_qty)"/>
+                        <span t-if="website.show_line_subtotals_tax_selection == 'tax_excluded'"
```

Comment:
Either you vertically align attributes between them or you use a fixed indentation.
I do not enforce one method over another, but please don't mix both :)

```xml
<span attribute-a="something"
      attribute-b="something"
      attribute-c="something"
      >

<span
    attribute-a="something"
    attribute-b="something"
    attribute-c="something"
    >